### DISCUSSION TITLE
Nicely worded and consistent package description

### DISCUSSION THREAD

**Discussion by Rowlando13** [NREL/mappymatch#150] [Ideas]
A one paragraph description for the project appears in multiple places: 
- README for the github repo 
- Pypi project description 
- Conda forge description 
- The landing page for the documentation  
I would like to make this nicely written and also very communicative about what the package does and does not do. Mappymatch fills a very niche need, so I feel like it is worth some effort so that it is easily found and also does not waste people's time.

**Comment by Rowlando13** [comment]
What I have so far: 

Mappymatch is a pure-python package developed and open sourced by the National Renewable Energy Laboratory. It contains a collection of "Matchers" that enable matching a GPS trace (series of GPS coordinates) to a map. 

The current and planned Matchers are: 

- `LCSSMatcher`: A matcher that implements the LCSS algorithm described in this [paper](https://doi.org/10.3141%2F2645-08). Works best with high resolution GPS traces.  
-  `OsrmMatcher`: A light matcher that pings an OSRM server to request map matching results. See the [official documentation](http://project-osrm.org/) for more info.
- `ValhallaMatcher` (planned): A matcher to ping a [Valhalla](https://www.interline.io/valhalla/) server for map matching results. 

Currently supported map formats are: 
- Open Street Maps

**Comment by Rowlando13** [comment]
I could use some verification and expansion @nreinicke .

**Reply by nreinicke** [reply]
I like this idea, just added some updates to the description above.

**Reply by Rowlando13** [reply]
Looks good. @machallboyd have any edits before I put it in all the places.
